app/src/main/java/org/akvo/flow/presentation/datapoints/map/offline/list/OfflineAreasListPresenter.java

```diff
@@ -132,4 +169,22 @@ public void onError(Throwable e) {
                 });
         disposables.add(subscribeWith);
     }
+
+    public void selectRegion(long regionId) {
+        Map<String, Object> params = new HashMap<>(2);
+        params.put(SaveSelectedOfflineArea.AREA_ID_PARAM, regionId);
+        saveSelectedOfflineArea.execute(new DisposableCompletableObserver() {
+            @Override
+            public void onComplete() {
```

Comment:
do we need the method if it's empty?

Reply:
when implementing an interface, one needs to implement all methods unfortunately